docs/source/intro/intro.rst

```diff
@@ -24,6 +24,8 @@ Here is a summary of the requirements, see `Ansible Installation <https://docs.a
 | Other   | SSH to Managed Machine  | Account with escalated privileges |
 +---------+-------------------------+-----------------------------------+
 
+To anyone new to ESGF and/or Web-based service deployments, running services in HTTPS is *a must*, and this requires SSL certificates recoginized by current browsers and other http(s) clients, eg. `wget, `curl`, Python `requests`.  If you haven't already, please check with your organization to see if you already purchase certificates from vendor.  If so, purchase one for your targeted ESGF Node server.  If not, free certificates can be obtained from ``LetsEncrypt``.  See the `Web Certificate Section <../usage/usage.html#web-certificate-installation>`_ within this site for more information.
```

Comment:
"`wget, `curl`, Python `requests`" -> "``wget``, ``curl``, Python ``requests``"

Reply:
Github formatting makes it hard to see. If you want a word monospaced and "blocked" you need 2 ticks on each side. Also you have mismatching ticks.

Reply:
Needs to be like the LetsEncrypt

Reply:
Also, could you give this section a header, like "SSL Certificates".